packages/swapkit/helpers/src/types/chains.ts

```diff
@@ -103,6 +106,8 @@ export enum BaseDecimal {
   OP = 18,
   THOR = 8,
   ZEC = 8,
+  // TODO do some research on this
+  RADIX = 8,
```

Comment:
https://dashboard.radixdlt.com/resource/resource_rdx1tknxxxxxxxxxradxrdxxxxxxxxx009923554798xxxxxxxxxradxrd/summary

This suggests Radix also has 18 decimals as default